RELEASES.md

```diff
@@ -16,17 +16,28 @@ part of Continuous Delivery (CD) nightly releases. See also the
 
 Table of contents:
 
-- [Installing releases using pip](#installing-releases-using-pip)
-  - [Installing ROCm Python packages](#installing-rocm-python-packages)
-  - [Using ROCm Python packages](#using-rocm-python-packages)
-  - [Installing PyTorch Python packages](#installing-pytorch-python-packages)
-  - [Using PyTorch Python packages](#using-pytorch-python-packages)
-- [Installing from tarballs](#installing-from-tarballs)
-  - [Installing release tarballs](#installing-release-tarballs)
-  - [Installing per-commit CI build tarballs manually](#installing-per-commit-ci-build-tarballs-manually)
-  - [Installing tarballs using `install_rocm_from_artifacts.py`](#installing-tarballs-using-install_rocm_from_artifactspy)
-  - [Using installed tarballs](#using-installed-tarballs)
-- [Using Dockerfiles](#using-dockerfiles)
+- [Releases](#releases)
+  - [Installing releases using pip](#installing-releases-using-pip)
+    - [Python packages release status](#python-packages-release-status)
+    - [Installing ROCm Python packages](#installing-rocm-python-packages)
+      - [gfx94X-dcgpu](#gfx94x-dcgpu)
+      - [gfx950-dcgpu](#gfx950-dcgpu)
+      - [gfx110X-dgpu](#gfx110x-dgpu)
+      - [gfx1151](#gfx1151)
+      - [gfx120X-all](#gfx120x-all)
```

Comment:
I could see leaving out some of these from the inlined table of contents. They are still there in GitHub's built-in "outline": 

<img width="1919" height="1154" alt="Image" src="https://github.com/user-attachments/assets/6caf264b-8b6c-4c8e-b4cf-cdddfb91280e" />

You do need to hunt for that though: 

<img width="260" height="134" alt="Image" src="https://github.com/user-attachments/assets/db693bc0-07d5-46cf-a4dc-ebfdc294986c" />